yccp/sweeps/namers.py

```diff
@@ -18,13 +18,30 @@
         "join",
     ]
 
+import hashlib
 import logging
 log = logging.getLogger(__name__.split(".")[0])
 
 from .. import utils
 rget = utils.get_recursive
 
 
+def create_custom(listOfPaths, name,
```

Comment:
Name does not reflect what it is doing.

You can already specify several namers for dictionary/final file name - why is this function needed?
If you want to extract and combine several strings it would be easier just to write a custom namer, as they are only functions (`ParameterSet` → `String`).